tutorials/misc/gles2_gles3_differences.rst

```diff
@@ -50,20 +50,20 @@ GLES2 is not capable of using High Dynamic Range (HDR) rendering features. If HD
 project, or for a given viewport, Godot will still user Low Dynamic Range (LDR) which limits
 viewport values to the ``0-1`` range.
 
-SpatialMaterial features
-------------------------
+StandardMaterial3D features
+---------------------------
 
-In GLES2, the following advanced rendering features in the :ref:`SpatialMaterial <class_SpatialMaterial>` are missing:
+In GLES2, the following advanced rendering features in the :ref:`StandardMaterial3D <class_StandardMaterial3D>` are missing:
 
 - Refraction
 - Subsurface scattering
 - Anisotropy
 - Clearcoat
 - Depth mapping
 
-When using SpatialMaterials they will not even appear in the editor.
+When using StandardMaterial3Ds they will not even appear in the editor.
 
-In custom :ref:`ShaderMaterials <class_ShaderMaterial>`, you can set values for these features but they
+In custom :ref:`class_StandardMaterial3D <class_StandardMaterial3D>`, you can set values for these features but they
```

Comment:
There are a few false positives like this one. ShaderMaterials still exist and are very different from StandardMaterial3Ds. So all references to ShaderMaterials should be reverted back.

Reply:
I've fixed it and double-checked that I haven't done it elsewhere too.